components/home/Features.vue

```diff
@@ -17,12 +17,12 @@
         />
       </v-stack>
       <v-stack justify="space-between" md-direction="column" md-align="center">
-        <v-stack
-          direction="column"
-          sm-direction="row"
-          class="features-description-container"
-        >
-          <v-stack justify="space-between" style="margin-top: 2rem">
+        <v-stack direction="column" class="features-description-container">
+          <v-stack
+            sm-direction="column"
+            justify="space-between"
+            class="blurb-container"
+          >
```

Comment:
Removed scrolling on x-axis. Instead features will come one after another in columnar fashion